_assets/stylesheets/modules/_header.scss

```diff
@@ -4,39 +4,33 @@
 
 header {
   display: block;
-  height: 500px;
-  padding: 0 15px;
+  padding: 125px 15px;
   text-align: center;
-  background: $brand-info;
+  background: linear-gradient(to top, #f46b45, #eea849);
```

Comment:
Color literals like `#f46b45` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#eea849` should only be used in variable declarations; they should be referred to via variable everywhere else.